Server/fn_startTraffic.sqf

```diff
@@ -55,7 +55,7 @@ private _activeVehiclesAndGroup = [];
       if (isMultiplayer) then {
          {
             if (isPlayer _x) then {
-               _allPlayerPositionsTemp = _allPlayerPositionsTemp + [position vehicle _x];
+               _allPlayerPositionsTemp = _allPlayerPositionsTemp append [position vehicle _x];

```

Comment:
Nooo not like that. append returns nil and modifies the original.
So you are doing `_allPlayerPositionsTemp = nil`
https://community.bistudio.com/wiki/append